lib/browserLanguageDetector.js

```diff
@@ -0,0 +1,36 @@
+const { i18n } = require("./../next-i18next-static-site.config");
+
+function browserLanguageDetector() {
+
+    // Converts languages like 'en-US' to 'en'
+    const convertLanguageCode = (code) => {
+        const baseLanguageCode = code.split('-')[0];
+        return baseLanguageCode;
+    };
```

Comment:
You can move that function outside the `browserLanguageDetector` but in this same file.

Reply:
Done!